pkg/operator/controller/status.go

```diff
@@ -53,6 +53,27 @@ func (r *reconciler) syncOperatorStatus() error {
 	if err != nil {
 		return fmt.Errorf("failed to get operator state: %v", err)
 	}
+
+	related := []configv1.ObjectReference{
+		{
+			Resource: "namespaces",
+			Name:     "openshift-ingress-operator",
```

Comment:
This should be `r.Namespace`, but we were already hard-coding it before, so it can likewise be filed under "namespace handling tech debt".